src/test/java/javax/xml/crypto/test/dsig/TestUtils.java

```diff
@@ -107,10 +107,10 @@ public class TestUtils {
     private static final String DSA_2048_X =
         "14562787764977288900757387442281559936279834964901963465277698843172";
     private static final String RSA_MOD =
-        "010800185049102889923150759252557522305032794699952150943573164381936603255999071981574575044810461362008102247767482738822150129277490998033971789476107463";
+        "25623882427526707322866322256240741413699766555508113361975182446922418415089438936415989175296088974016615587740736574241342957393002320762010474925830839106425586207448011995748812345401230782267753189386103568502885279593734636842405461656752755702912746172339027034096681889413854955725588164346940842786630738833311668086338812048993406627875818948914819316678433784472883107947150827418112730771165726422295838065290136656204305183681062358941954544617538033203389194497917362344779507907933457930952778285948136508570166989856730778407368193764241294623598058775696847518736534397297112966951721385759730339389";
```

Comment:
I'm ok with changing this to 2048. However, you also need to update all the calls to `TestUtils.getPublicKey("RSA", 512)` to `TestUtils.getPublicKey("RSA", 2048)` (and `getPrivateKey("RSA", 512)`. There are a few instances in `TestUtils` and a couple in `XMLSignatureTest`.

You also need to change line 159 to:

`            if (keysize == 2048) {`

Reply:
ah, right. will do.